backend/src/database/migrations/20221012051800-add-api-key-hash-column-to-user.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('users', 'api_key_hash', {
+      type: Sequelize.DataTypes.STRING(255),
+      allowNull: true,
+    })
+    await queryInterface.sequelize.query(
+      'UPDATE users SET api_key_hash = api_key;'
+    )
+    await queryInterface.createFunction(
+      'format_user_email',
+      [],
+      'trigger',
+      'plpgsql',
+      `
+      NEW.email = LOWER(NEW.email);
```

Comment:
I think u forgot to modify this trigger copied from another migration?

Reply:
I'm not sure what you mean, but the omission of any reference to `api_key` is intentional! See [here](https://github.com/opengovsg/postmangovsg/pull/1841/files#r994095300).